cmd/gaia/cli_test/test_helpers.go

```diff
@@ -426,11 +426,18 @@ func (f *Fixtures) QueryGovParamTallying() gov.TallyParams {
 }
 
 // QueryGovProposals is gaiacli query gov proposals
-func (f *Fixtures) QueryGovProposals(flags ...string) string {
+func (f *Fixtures) QueryGovProposals(flags ...string) gov.Proposals {
 	cmd := fmt.Sprintf("gaiacli query gov proposals %v", f.Flags())
 	stdout, stderr := tests.ExecuteT(f.T, addFlags(cmd, flags), "")
+	if strings.Contains(stderr, "No matching proposals found") {
```

Comment:
Thanks @jackzampolin -- we should use error variables in the future (e.g. `strings.Contains(stderr, gov.ErrNoMatchingProposals)`

Reply:
Absolutely. There is a bunch more that needs to be done to standardize errors across the various client software.